react/features/follow-me/subscriber.js

```diff
@@ -12,13 +12,12 @@ import { FOLLOW_ME_COMMAND } from './constants';
 /**
  * Subscribes to changes to the Follow Me setting for the local participant to
  * notify remote participants of current user interface status.
- *
- * @param sharedDocumentVisible {Boolean} {true} if the shared document was
- * shown (as a result of the toggle) or {false} if it was hidden
+ * Changing newSelectedValue param to off, when feature is turned of so we can
+ * notify all listeners.
  */
 StateListenerRegistry.register(
     /* selector */ state => state['features/base/conference'].followMeEnabled,
-    /* listener */ _sendFollowMeCommand);
+    /* listener */ (newSelectedValue, store) => _sendFollowMeCommand(newSelectedValue || 'off', store));
```

Comment:
Style wise, you can use a default argument of 'off' for _sendFollowMeCommand  instead of using a fallthough ||.

Reply:
There are multiple listeners using the same function. 
Using another function is not an option, as we need to duplicate the checks about moderator and stuff.

Reply:
I disagree that using another function is not an option; it is completely an option and would remove a conditional from the function below. Plus a new function would dis-empower the value of "off", which I would say does not inherently communicate that it is exclusive to enabling or disable follow me. However, I'm not gonna fight for the other function.

Reply:
But for the rest of the checks, I need to create one more function, so they will come up to two or duplicate them?